test/controllers/api/base_controller_test.rb

```diff
@@ -3,18 +3,23 @@
 describe Api::BaseController do
 
   it 'should show entrypoint' do
-    get(:entrypoint, { api_version: 'v1', format: 'json' } )
+    get :entrypoint, api_version: 'v1', format: 'json'
     assert_response :success
   end
 
   it "determines show action options for roar" do
     @controller.class.resource_representer = "rr"
-    @controller.show_options[:represent_with].must_equal "rr"
+    @controller.send(:show_options)[:represent_with].must_equal "rr"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.